src/event/repository.jl

```diff
@@ -0,0 +1,70 @@
+"""
+    event_repository()
+Repository information as a Dict
+"""
+event_repository() = EVENT["repository"]
+
+"""
+    event_repository_name()
+output is like: `"GitHubActions.jl"`
+"""
+event_repository_name() = event_repository()["name"]
+
+"""
+    event_repository_full_name()
+output is like: `"aminya/GitHubActions.jl"`
+"""
+event_repository_full_name() = event_repository()["full_name"]
+
+"""
+    event_repository_html_url()
+HTML url as String like `"https://github.com/aminya/GitHubActions.jl"`
+"""
+event_repository_html_url() = event_repository()["html_url"]
+
+"""
+    event_repository_git_url()
+git url as String like `"git://github.com/aminya/GitHubActions.jl.git"`
+"""
+event_repository_git_url() = event_repository()["git_url"]
+
+"""
+    event_repository_ssh_url()
+ssh url as String like `"git@github.com:aminya/GitHubActions.jl.git"`
+"""
+event_repository_ssh_url() = event_repository()["ssh_url"]
+
+"""
+    package_spec = event_packagespec()
+
+This function returns the repository PackageSpec.
+"""
+function event_packagespec()
+    url = event_repository_html_url()
+    rev = event_rev()
+    package_spec = PackageSpec(url = url, rev = rev)
```

Comment:
I just tested this in a pull request. If we use the default settings of `@actions/checkout`, the URL will be the URL of the target repository, which is not what we want. We should run the action for the source repository.

rev is correct since I find it using `event_rev`

Reply:
@christopher-dG Any idea about this?